src/modules/pages/page_repository.ts

```diff
@@ -58,4 +59,16 @@ export namespace Page {
 
     return query
   }
+
+  export const destroyFile = async (id: number) => {
+    const query = Files().where('id', id).delete()
```

Comment:
Type vriable?

Reply:
Kalau di query mah susah type data dari knex mah kang, jadi biarkan bawaan knex

Reply:
aku sederhana in lagi kang, cc87379